x/concentrated-liquidity/LICENSE.md

```diff
@@ -0,0 +1,14 @@
+
+Copyright 2023 osmosis-1
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the “Software”), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+1) x/concentrated-liqudity is for use exclusively on osmosis-1, the cosmos blockchain that:
```

Comment:
Oh uh have you seen anyone else do something like this? Not sure if this 1. is a desirable constraint to add and 2. would actually prevent the behavior it seems to be trying to prevent

Reply:
I did it, but this isn't court tested, it's the result of a conversation with Sunny.